packages/core/src/components/dialog/multistepDialog.tsx

```diff
@@ -233,30 +234,63 @@ export class MultistepDialog extends AbstractPureComponent2<MultistepDialogProps
         if (this.state.selectedIndex > 0) {
             const backButtonProps = steps[selectedIndex].props.backButtonProps ?? this.props.backButtonProps;
 
-            buttons.push(
+            if (backButtonProps?.tooltipContent) {
```

Comment:
please avoid implicit type coercion here. instead use an explicit check:

```suggestion
            if (backButtonProps?.tooltipContent != null) {
```